x-pack/plugins/lens/public/types.ts

```diff
@@ -470,6 +470,7 @@ export type VisualizationDimensionGroupConfig = SharedDimensionProps & {
   supportsMoreColumns: boolean;
   /** If required, a warning will appear if accessors are empty */
   required?: boolean;
+  requiredMinDimensionCount?: number;
```

Comment:
I think I wasn't clear enough with my previous comment, I'm sorry! What I meant was to remove `required` property and replace it by using the new `requiredMinDimensionCount` - and just set it on 1 for all the required dimensions in all the visualizations. You can decide to leave it as it is here and come back to it separately. Right now it's just duplicated info here.

Reply:
Let's leave `requiredMinDimensionCount` and` required` for now. I changed the code a bit and now `requiredMinDimensionCount` works without` required`. I think we will be able to do this in the future if we understand that one field will be enough.